scripts/make.go

```diff
@@ -255,7 +255,7 @@ func testFlags() []string {
 	if err != nil {
 		log.Fatal(err)
 	}
-	testFlags := []string{"-count", "1", "-p", "1"}
+	testFlags := []string{"-count", "1", "-p", "1", "-timeout", "0"}
```

Comment:
Having a timeout set for the tests is very useful and 10 minutes is a reasonable default, if you need infinite timeouts add a new flag to make.go to enable those manually.

Reply:
modified.